wine-tkg-git/wine-tkg-profiles/chaotic-staging.cfg

```diff
@@ -6,6 +6,6 @@ _proton_fs_hack="true"
 _FS_bypass_compositor="true"
 _win10_default="true"
 _protonify="true"
-_community_patches="amdags.mypatch ntdll_Map_top-down_if_dll_characteristics_include_DYNAMIC_BASE.mypatch 0002-proton_LFH.mypatch 0002-proton_QPC.mypatch mfplat_nv12_d3d11_buffers.mypatch"
+_community_patches="roblox_mouse_fix.mypatch"
```

Comment:
No need to drop older patches from it (the `community_patches` is a space-separated list).

@Tk-Glitch, does "roblox_mouse_fix" break any other game?

Reply:
nvm opened new one